### PR TITLE
Feature/tests

### PR DESCRIPTION
**Pull request by YoseptF**
We made the following test with jest:

- The gameController should
    -  ✓ be an object (3ms)
    - play with the selected square and
      - ✓ should fail if the square is already taken (16ms)
      - ✓ should succeed if the square is not already taken (4ms)
      - ✓ should win if you have 3 in a row (6ms)
      - ✓ should draw if nobody won (14ms)
    - start the game
      - ✓ display error message if players are null (2ms)
      - ✓ display player turn if players inputs are not null (2ms)

### CODE REVIEW

File: test/gameController.test.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+      gameBoard.squares = [
+        { innerText: 'x' },
+        { innerText: 'x' },
+        { innerText: '' },
+      ];
+      gameController.brd = gameBoard;
+      gameController.play(2);
+      const winMessage = document.querySelector('.winMessage p');
+      expect(winMessage.innerHTML).toEqual('Congratulations Yair, you won!!!');
+    });
```
Comment (maelfosso) [inline test/gameController.test.js:75]: The `win` method returns a boolean.
Write according to that not according the message. You except the win method to return either true or false.

**Review by maelfosso — CHANGES_REQUESTED** [review]
# Changes required :red_circle: 

Hello @YoseptF @YairFernando67 
Everything is almost OK. 
Just a few thing to fix before moving forward.

Kindly read the comment left below and make the suggested changes then submit a new code review request.
Thanks

Mael FOSSO

**Comment by YoseptF** [comment]
We added 2 more tests, now we are testing the `win` method, it returns false when someone won, and true if the game continues :D

**Review by jofer86 — APPROVED** [review]
## :wave: :wave: Hello!, I'm Jorge, and  I'm here to review this Pull Request, let's take a look shall we :eye: :eye: :eye: 
- Changes marked as **[OPTIONAL]** are not hard requirements but are nice to have.
- All the required changes were tackled!! Great job!!
# `STATUS: APPROVED` :rocket: :+1: :rocket: 
--------------------------------------------------------------------
Cheers, 
Jorge Fernando Rincón Figueroa.
Technical Support Engineer.
**Slack: @jofer86**

**Pull request merged**